src/libraries/System.Numerics.Vectors/System.Numerics.Vectors.sln

```diff
@@ -26,30 +30,27 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{7A13467C-8469-4562-B00F-C4D61E8A1855}"
 EndProject
 Global
-	GlobalSection(NestedProjects) = preSolution
-		{634A3B2B-09F5-4810-B630-ADE4D36C47DF} = {D47BF410-2A81-48E2-A7E5-CE29794B572C}
-		{ED450846-85A0-4CED-B4D9-9EB769CF794B} = {D47BF410-2A81-48E2-A7E5-CE29794B572C}
-		{7FAD5E59-D362-4ED6-B3D5-FAD9CD5A2598} = {D47BF410-2A81-48E2-A7E5-CE29794B572C}
-		{9EDBE037-EFE0-4B72-B602-E4C3F0C05F2F} = {453F2E85-9FFF-4833-BB86-1B8E73191756}
-		{B38797B1-BB45-4B30-9D4F-79D9F4B3735B} = {453F2E85-9FFF-4833-BB86-1B8E73191756}
-		{9805AA59-A72D-4D20-BBCE-EBD90DC7840A} = {7A13467C-8469-4562-B00F-C4D61E8A1855}
-		{5B2027FA-F43A-4E80-880F-B3A7A2720AA7} = {7A13467C-8469-4562-B00F-C4D61E8A1855}
-		{DFC21F4E-EC4F-4310-A4DA-B7094AA4D237} = {7A13467C-8469-4562-B00F-C4D61E8A1855}
-		{875DAA4F-1BC0-4EE2-9A04-EDD2FC5C773F} = {7A13467C-8469-4562-B00F-C4D61E8A1855}
-		{BD5C16FE-CAAA-45CB-91F3-C66A2493C518} = {7A13467C-8469-4562-B00F-C4D61E8A1855}
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\System.Private.CoreLib\src\System.Private.CoreLib.Shared.projitems*{634a3b2b-09f5-4810-b630-ade4d36c47df}*SharedItemsImports = 5
```

Comment:
I don't understand why this is being added? Did you change this on VS?

Reply:
No, I didn't modify the sln file myself. Visual Studio did this. I'll revert the sln file change.

Reply:
@safern reverted.